src/Model/ContactManager.php

```diff
@@ -22,4 +22,27 @@ public function insert(array $user): int
         $statement->execute();
         return (int)$this->pdo->lastInsertId();
     }
+
+    public function selectOneById(int $id): array
+    {
+        $query = "SELECT * FROM " . static::TABLE . " WHERE id = $id;";
+        $statement = $this->pdo->query($query);
+        $userMessage = $statement->fetch();
+
+        return $userMessage;
+    }
```

Comment:
I don't see why you redefined this method, specially when the one from AbstractManager is more secure because here you should prepare your query. $id can't be trusted

Reply:
Absolutely, method in AbstractManager will perfectly do the job.
This will be corrected in next PR.